.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'prettier',
     'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
-    'plugin:import/warnings'
+    'plugin:import/warnings',
```

Comment:
**Minor detail**: Is this trailing comma needed?

Reply:
Yup it's an ESLint rule, I didn't add it, it was added automatically by it